docs/main_text.md

```diff
@@ -437,7 +437,15 @@ We then quantile normalize all samples as described above.
 
 We've made our analyses underlying processing choices and exploring test compendia available at our <a href = "https://github.com/AlexsLemonade/compendium-processing" target = "blank">`compendium-processing`</a> repository.
 
-### Download Folder
+### Collapsing by genus
+
+Microarray platforms are generally designed to assay samples from a specific species.
+In some cases, samples that are not from the species for which a microarray platform is specifically designed for are in publicly available data surveyed by refine.bio (e.g., samples labeled _Bos indicus_ will be run on _Bos taurus_ microarrays or mouse crosses that are not labeled _Mus musculus_ will be run on _Mus musculus_ microarrays). 
+When we encounter this in refine.bio, we will include samples from the species that differs from the platform species when the two species share a genus in the larger species' normalized compendium (e.g., _Bos indicus_ samples run on _Bos taurus_ microarrays are included in the _Bos taurus_ normalized compendium).
```

Comment:
```suggestion
When we encounter this in refine.bio, we will include samples in a compendium from species that differ from the primary platform species when the two species share a genus (e.g., _Bos indicus_ samples run on _Bos taurus_ microarrays are included in the _Bos taurus_ normalized compendium).
```

My change loses the directionality of the the larger species, but I replaced that with "primary" which I hope is clear enough?